src/driver/sqljs/SqljsDriver.ts

```diff
@@ -53,6 +55,9 @@ export class SqljsDriver extends AbstractSqliteDriver {
      */
     async connect(): Promise<void> {
         this.databaseConnection = await this.createDatabaseConnection()
+        this.version = this.databaseConnection.exec(
```

Comment:
`await` isn't needed?

Reply:
@pleerock [sql.js docs](https://github.com/sql-js/sql.js/) claim it's synchronous.